guide/how-it-works/sign-in-with-bitcoin.md

```diff
@@ -0,0 +1,161 @@
+   retina = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/direct-connection@2x.png"
+   alt-text = "Browser window with a wallet extension installed"
+   width = 800
+   height = 411
+%}
+
+#### Bridged communication
+
+There is more friction if the user needs to be involved in this communication. However, this extra friction provides more user control and better security. In the visualization below, the service and user wallet cannot directly communicate with each other. Instead, the web application presents instructions as QR codes that the user needs to scan with their (mobile) wallet. The wallet then sends instructions to the application server, which updates the web application.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/bridged-connection.png"
+   retina = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/bridged-connection@2x.png"
+   alt-text = "Communication flow between a browser window and a smart phone wallet"
+   width = 800
+   height = 411
+%}
+
+This does not necessarily have to be done via QR code. The most basic method could even involve manually copying and pasting messages and identifiers back and forth.
+
+## Signing in
+
+The biggest hurdle at this stage is that most users will not be familiar with this authentication method and are not likely to have a wallet they can readily use. Ensure that there are concise explanations and tips to get users started.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/signin-info.png"
+   retina = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/signin-info@2x.png"
+   alt-text = "Sign in screen with tips for wallets to use and general information"
+   width = 800
+   height = 411
+%}
+
+Authentication is a simple process in which the service sends a message to the wallet. The wallet then uses cryptography to generate a unique identifier from the message, which it sends back to the service. This identifier can then be used to establish an account.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/signin-request.png"
+   retina = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/signin-request@2x.png"
+   alt-text = "Browser window with a modal for approving web service authentication"
+   width = 800
+   height = 411
+%}
+
+Keep in mind that your service will not be able to retrieve user names, emails, balances, transactions, or other information from the wallet. Only the minimal information to identify the user is communicated. If you need to communicate with users about their accounts or activity, make sure you request an email or other contact point.
+
+## Signing out
+
+Ensuring users are signed out (manually or via session time-outs) is important. The account may have payment allowances, or other problematic permissions, that could otherwise be abused by third parties.
```

Comment:
An image for this would be handy.

Reply:
What would the image show? Doesn't seem like any unique design patterns that don't exist in any other application.